airbyte-webapp/src/scss/global.scss

```diff
@@ -1,3 +1,4 @@
+@use "./customProperties";
```

Comment:
Super minor nitpick, but could we maybe name this file `palette` or  `theme-light`, so it's more intuitive finding it if you look for it :)

Reply:
Related: I chose to use dashed-names over camelCase for SCSS files as it is the typical SCSS convention.

Reply:
Sure! Renamed it to `_theme-light.scss` now